packages/layout/src/grid-layout/border-helper.js

```diff
@@ -0,0 +1,53 @@
+import {
+    TOP, BOTTOM, LEFT, RIGHT, CENTER, BLANK_BORDERS, COLUMN, ROW, VIEW_INDEX
+} from '../enums/constants';
+
+const borderMap = {
+    '00': null,
+    '01': COLUMN,
+    '02': null,
+    10: ROW,
+    11: CENTER,
+    12: ROW,
+    20: null,
+    21: COLUMN,
+    22: null
+};
```

Comment:
Map having different key types.
Instead of Numbers we have give specific name for each as used by Layout

Reply:
Use of null, should me some meaning full idetifiers, replace null with some constant.